models/object_detection_nanodet/coco.names

```diff
@@ -0,0 +1,80 @@
+person
```

Comment:
This file is not needed if we keep these classes in the wrapper class.

Reply:
- Resolved the changes and replaced the file with classes in the demo.py file
- Removed file 'coco.names'